src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In JavaScript, which property of an element allows you to get or set the HTML markup contained within the element?",
+    Answer: "innerHTML",
+    Distractor1: "insertAdjacentHTML",
+    Distractor2: "insertElement",
+    Distractor3: "setHTML",
+    Explanation: "The innerHTML is a property of the Element that allows you to get or set the HTML markup contained within the element.",
```

Comment:
Element doesn't need to be capitalized here

```suggestion
    Explanation: "The innerHTML is a property of the element that allows you to get or set the HTML markup contained within the element.",
```